taxi/urls.py

```diff
@@ -8,7 +9,9 @@
     path("cars/", CarListView.as_view(), name="car_list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car_detail"),
     path("drivers/", DriverListView.as_view(), name="driver_list"),
-    path("drivers/<int:pk>/", DriverDetailView.as_view(), name="driver_detail")
+    path("drivers/<int:pk>/", DriverDetailView.as_view(), name="driver_detail"),
+    path("/login/", views.LoginView.as_view),
+    path("/logout/", views.LogoutView.as_view),
```

Comment:
You don't need this if you use django.contrib.auth.urls. 

And you should add screenshots

Reply:
Tnx, new for me